docs-src/content/getting-started/streaming.md

```diff
@@ -3,68 +3,73 @@ title: "Streaming Recognition"
 weight: 30
 ---
 The following example shows how to transcribe an audio file using Cubic’s 
-Streaming Recognize Request. The example uses a WAV file as input to the 
-streaming recognition. We will query the server for available models and
-use the first model to transcribe the speech.
+Streaming Recognize Request. The stream can come from a file on disk or be
+directly from a microphone in real time.
 
 <!--more-->
 
+### Streaming from a local file
+
+The examples below use a WAV file as input to the streaming recognition.
```

Comment:
Here I think we should list again the supported file types - it's hidden in the intro but I think is worth repeating here

Reply:
We should probably add the same list to the Synchronous Recognition page.

Reply:
I will add a list of types to the `streaming.md` file. If I recall correctly, we don't support the same list of formats for Synchronous Recognition (we expect raw samples currently). [See Issue](https://github.com/cobaltspeech/cubicsvr/issues/51)